default.json5

```
@@ -77,22 +77,64 @@
       ],
     },
     {
-      groupName: "all major dependencies",
-      groupSlug: "all-major",
+      matchManagers: [
+        "gradle"
```

```suggestion
        "gradle",
```
please always use trailing commas in json5 files